src/components/Video/NotRecording.js

```diff
@@ -8,21 +8,23 @@ import { localised } from '../../locales'
 import type { LocalisedType } from '../../locales'
 
 type Props = {
-  hasError: boolean,
+  hasTimeoutError: boolean,
+  hasCameraError: boolean,
   onTimeout: void => void,
   onStart: void => void,
 } & LocalisedType
 
-const NotRecording = ({ translate, onStart, hasError, onTimeout }: Props) => (
+const NotRecording = ({ translate, onStart, hasTimeoutError, hasCameraError, onTimeout }: Props) => (
   <div>
-    { !hasError && <Timeout key="notRecording" seconds={ 12 } onTimeout={ onTimeout } /> }
+    { !hasTimeoutError && !hasCameraError ? <Timeout key="notRecording" seconds={ 12 } onTimeout={ onTimeout } /> : null }
     <div className={style.actions}>
       <div className={classNames(style.captureActionsHint, style.recordAction)}>
         { translate('capture.liveness.press_record') }
       </div>
       <button
         aria-label={translate('accessibility.start_recording')}
         className={classNames(style.btn, style.startRecording)}
+        disabled={hasCameraError}
```

Comment:
The recording button should not be clickable if recording took too long. Will this handle this case? I think the current behaviour does not allow you to click on the recording button if recording took too long

Reply:
No, this only disables when camera is actually unavailable. Any reason for disabling in the state of a person taking too long to shoot? I don't see this as being a case, unless it's already recording a video?

Reply:
@stefaniacardenas After chatting with @seewah I understand what is intended here, and made a fix. Now for videos:
- on "recording" component interaction is disabled when any error (timeout/camera) happens
- on "not recording" component interaction is disabled not only when camera error happens but also when recording is taking too long (hence it is still enabled when "camera became inactive" occurs, this is needed behaviour)

Reply:
Perfect!